benchmarks/DeepLearning/Ops/BatchMatMulInt32Op/CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+if (CROSS_COMPILE_RVV)
+  set(RISCV_GNU_TOOLCHAIN ${BUDDY_MLIR_BUILD_DIR}/thirdparty/riscv-gnu-toolchain)
+  set(RISCV_GNU_TOOLCHAIN_SYSROOT ${RISCV_GNU_TOOLCHAIN}/sysroot)
+  set(BUDDY_OPT_TRIPLE riscv64)
+  set(BUDDY_OPT_ATTR +v,+m)
+endif()
```

Comment:
Please remove these redundant cmake settings for they have appeared in buddy-benchmark/CMakeLists.txt